spec/models/vote_spec.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe Vote do
```

Comment:
you may want to check [`validate_uniqueness_of`](https://www.rubydoc.info/github/thoughtbot/shoulda-matchers/master/Shoulda%2FMatchers%2FActiveRecord:validate_uniqueness_of) instead :wink: